.github/workflows/release.yml

```diff
@@ -30,4 +29,17 @@ jobs:
           version: latest
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.GORELEASER_GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GORELEASER_GITHUB_TOKEN }}
+
+      - name: make release
+        run: |
+          make release
+          
+      - name: Update latest release
+        uses: "marvinpinto/action-automatic-releases@latest"
+        with:
+          repo_token: "${{ secrets.GITHUB_TOKEN }}"
+          prerelease: false
+          automatic_release_tag: latest/svix
+          files: |
+            svix.tar.gz
```

Comment:
We use goreleaser's official action to update our releases so I think this can be removed if you add the snapcraft file to line 30 `args: release --rm-dist` or just move the snapcraft configuration from `snapcraft.yml` to our  `.goreleaser.yml` file

Reply:
On the terms of generating an unversioned tar file for ease of installment -- Im not against it but we should still add `linux` to the file.  We can do this via another goreleaser step and save us from having to import another action.

We would want to modify the replacements section here`https://github.com/svix/svix-cli/blob/2d953fbdd9ef7e6f0872d07a614b32e6ec66bc3e/.goreleaser.yml#L17` or create a completely separate goreleaser step for Linux with the `build` name set to a constant value. See: https://goreleaser.com/customization/build/ for more details